### ISSUE TITLE
expect 0 to return 零元整

### ISSUE THREAD

**Issue by princemaple** [Jon-Millent/nomoney#5]
currently it doesn't return anything

**Comment by Jon-Millent** [comment]
I don’t have this problem here, I input 0, and it returns "零元整"

```js
import nomoney from "nomoney"

console.log("test return")
console.log(nomoney.stringify('0'))
```
![image](https://user-images.githubusercontent.com/17584565/146360048-5d19807a-52fa-4ef7-b00e-4050b359a87e.png)

**Comment by princemaple** [comment]
:) stringify allows an integer.

**Comment by princemaple** [comment]
`0` is blocked by `    if (!rmb) {`

Good to know `'0'` works though. Thanks!

**Comment by Jon-Millent** [comment]
Okay, I know this bug, I will fix it

**Comment by Jon-Millent** [comment]
I have fixed this bug in version 1.0.3. upgrading your dependencies can solve this problem.

**Comment by princemaple** [comment]
Thanks!
